github/repos_deployments.go

```diff
@@ -82,6 +82,26 @@ func (s *RepositoriesService) ListDeployments(owner, repo string, opt *Deploymen
 	return *deployments, resp, err
 }
 
+// GetDeployment returns a single deployment of a repository.
+//
+// GitHub API docs: https://developer.github.com/v3/repos/deployments/
```

Comment:
Can you please update it to point to the exact endpoint in the docs?

Actually, see https://github.com/google/go-github/issues/482#issuecomment-264770261, unless I'm missing it, it seems these endpoints are not documented.

If so, we should mention that (because if they're not documented, they may change or get removed without warning... unlikely, but possible).